telehealth-example/src/main/java/com/pubnub/components/example/getting_started/ui/view/Login.kt

```diff
@@ -0,0 +1,209 @@
+package com.pubnub.components.example.getting_started.ui.view
+
+import android.content.Context
+import android.content.Intent
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat.startActivity
+import com.pubnub.components.data.message.asMap
+import com.pubnub.components.example.getting_started.ChannelActivity
+import com.pubnub.components.example.getting_started.DefaultDataRepository
+import com.pubnub.components.example.getting_started.R
+
+object Login {
+    @Composable
+    fun View(
+        context: Context,
+        defaultDataRepository: DefaultDataRepository
+    ) {
+        var login by rememberSaveable { mutableStateOf("") }
+        val pass by rememberSaveable { mutableStateOf("") }
+        Box {
+            var visible by remember { mutableStateOf(false) }
+            Column(
+                modifier = Modifier.fillMaxSize(),
+                verticalArrangement = Arrangement.Top
+            ) {
+                Image(
+                    modifier = Modifier
+                        .padding(top = 50.dp)
+                        .size(140.dp)
+                        .align(Alignment.CenterHorizontally),
+                    painter = painterResource(id = R.drawable.logo),
+                    contentDescription = "logo"
+                )
+                Text(
+                    text = stringResource(R.string.login),
+                    style = TextStyle(fontSize = 24.sp, fontWeight = FontWeight.Bold),
+                    modifier = Modifier.padding(start = 24.dp, bottom = 24.dp)
+                )
+                Text(
+                    text = stringResource(R.string.username),
+                    style = TextStyle(fontSize = 16.sp),
+                    modifier = Modifier.padding(start = 24.dp)
+                )
+                OutlinedTextField(
```

Comment:
It should be a single-line input, password too. Additionally, there's no IME action on keyboard send button click

Reply:
Done